OpenSim/Tools/Test/test_wrapAllVis.osim

```diff
@@ -0,0 +1,745 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
The model is not used in this PR, and planned for WrapObject visualization test, it felt harmless at the time but I can remove it if it is confusing.